packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css

```diff
@@ -186,7 +187,9 @@
 
 .ag-charts-toolbar__button.ag-charts-toolbar__button--active {
     background: var(--ag-charts-toolbar-active-background-color);
+    border: 1px solid var(--ag-charts-toolbar-active-color);
```

Comment:
`border-color: var(--ag-charts-toolbar-active-color)` in case we change the other border styles